airflow/providers/apache/hive/example_dags/example_twitter_dag.py

```diff
@@ -77,8 +76,9 @@ def transfer_to_db():
         'retries': 1,
     },
     schedule_interval="@daily",
-    start_date=days_ago(5),
+    start_date=datetime(2021, 1, 1),
     tags=['example'],
+    catchup=False,
```

Comment:
Why this change?

Reply:
Only controlling the number of DAG runs that spin when users first enable the DAG now that we're transitioning to a static `start_date` value as best practice.